templates/resources/resource_detail.html

```diff
@@ -7,7 +7,7 @@
 
 {% block header_content %}
   {% include 'includes/header.html' with action_buttons='true' %}
-  {{ block.super }}
+  {% include "includes/messages.html" %}
```

Comment:
Can you make sure we leave messages? As this shows when the user edits their resource?